ov/ov_test.go

```diff
@@ -33,22 +33,22 @@ type OVTest struct {
 }
 
 // get Environment
-func (ot *OVTest) GetEnvironment() {
+func (ot *OVTest) GetEnvironment(env string) {
 	if os.Getenv("ONEVIEW_TEST_ENV") != "" {
 		ot.Env = os.Getenv("ONEVIEW_TEST_ENV")
 		return
 	}
-	ot.Env = "dev"
+	ot.Env = env
```

Comment:
So at the time, it seemed like this was server_profile test specific. So when I was adding new tests I added tests specific to other things. It keeps the test chunks small and specific to the tests we are running. I might have misunderstood this though. My test data looks like 
```
{
  "name": "dev"
  //initial test data
},
{
  "name": "test_ethernet_network",
  // data for ethernet
.
.
.
```